### DISCUSSION TITLE
Remember panel height

### DISCUSSION THREAD

**Discussion by the-baaron** [alefragnani/vscode-bookmarks#383] [Ideas]
Hi! First of all, thanks for making my life a lot easier! I love this extension.

Unfortunately, when I start VS-code the extension takes half of the Explorer column. It would be amazing to just have it at the bottom, and it would remember it's size, so that you don't have to make it smaller every time VS-code starts

This is the current behaviour: 
![Screenshot 2021-02-11 at 09 55 51](https://user-images.githubusercontent.com/6582045/107617135-9d2b2a00-6c4f-11eb-8877-4f366ec36a41.png)

**Comment by alefragnani** [comment]
Hi @Ronald-Baars ,

First of all, thank you for your kind words. I'm glad to know the extension is useful 👍 .

About the panel height, this is controlled by VS Code itself, not the extension. The only attribute the extension can control is to define if the extension starts with the panel collapsed or not. Other than that, is VS Code which will take care. Even the save/restore positions when you close/open each VS Code window.

Looking quickly at VS Code issues, I didn't find anything about it. Maybe it wasn't noticed yet, so I suggest you to open an issue in VS Code itself, so they could take care of. Maybe there is an issue on _user contributed/moved panels_ which is raising this behaviour.

Hope this helps

**Reply by the-baaron** [reply]
Hmm, interesting! I'll do that then! Thanks!
